libr/core/cmd_anal.c

```diff
@@ -3873,6 +3866,7 @@ static int cmd_anal_all(RCore *core, const char *input) {
 		"aat", " [len]", "analyze all consecutive functions in section",
 		"aap", "", "find and analyze function preludes",
 		"aav", " [sat]", "find values referencing a specific section or map",
+		"aau", " [len]", "list mem areas (larger than len bytes) not covered by functions",
```

Comment:
I chose **aau** because I thought that it was kind of the counterpart of **aai** (one providing info about
the analyzed bytes and one about the ones that are not). However, I see your point about **aa**
actually doing the analysis. So, should it be something like **afu**?